var/spack/repos/builtin/packages/git/package.py

```diff
@@ -23,6 +23,11 @@ class Git(AutotoolsPackage):
     #       https://www.kernel.org/pub/software/scm/git/git-manpages-{version}.tar.gz
     # You can find the source here: https://mirrors.edge.kernel.org/pub/software/scm/git/sha256sums.asc
     releases = [
+        {
+            'version': '2.28.0',
+            'sha256': 'f914c60a874d466c1e18467c864a910dd4ea22281ba6d4d58077cb0c3f115170',
+            'sha256_manpages': '665a7fa41f9a39248e48422dcf533e1a0133a4a6eddfc25ee55e28e42fc07cb7'
```

Comment:
The checksum for the manpages seems to be from the tar.xz not the tar.gz that's needed.
```
==> Installing git
==> No binary for git found: installing from source
==> Error: ChecksumError: sha256 checksum failed for /tmp/aweits/spack-stage/resource-git-manpages-wn5r3etaep3vxewhdq5cynyr7nnwl7df/git-manpages-2.28.0.tar.gz
    Expected 665a7fa41f9a39248e48422dcf533e1a0133a4a6eddfc25ee55e28e42fc07cb7 but got 3cfca28a88d5b8112ea42322b797a500a14d0acddea391aed0462aff1ab11bf7

```

Reply:
Replaced with correct checksum.